pkg/handler/crud.go

```diff
@@ -237,6 +237,10 @@ func (c *crud) PutFlag(params flag.PutFlagParams) middleware.Responder {
 		f.Notes = *params.Body.Notes
 	}
 
+	if err := validateFlagIfEnabled(f); err != nil {
```

Comment:
Is it possible there is `enabled` is being set to true in params? like so https://github.com/checkr/flagr/blob/cff1f323fd82b12ab2792ce4b2c3fd49f3e30cce/swagger_gen/models/put_flag_request.go#L30